public/partials/membership_card.html

```diff
@@ -18,7 +18,7 @@ <h3 class="white">Membership</h3>
         <div flex layout="row">
             <md-icon class="no-margin" md-svg-icon="/images/icons/icons_farmer_active.svg"></md-icon>
             <div layout-align="center" style="text-align: center;" layout="column" ng-class="{'md-offset': 1}">
-                Active
+                {{isActive(memberships[0].mi_expiration.split("T")[0])?  "Active" :  'Inactive'}}
```

Comment:
We have mentioned this before.  It is important that you NOT manipulate the date yourself by doing the `split()` to separate the time from the date.  This potentially creates issues with the date in our region.